server/persistence/memory/memory.go

```diff
@@ -1,13 +1,54 @@
 package memory
 
 import (
+	"sync"
+
 	"github.com/joshprzybyszewski/cribbage/server/persistence"
 )
 
+var _ persistence.DB = (*memDB)(nil)
+
+type memDB struct {
+	persistence.ServicesWrapper
+
+	lock sync.Mutex
+}
+
 func New() persistence.DB {
-	return persistence.New(
+	sw := persistence.NewServicesWrapper(
 		getGameService(),
 		getPlayerService(),
 		getInteractionService(),
 	)
+
+	return &memDB{
+		ServicesWrapper: sw,
+	}
+}
+
+func (mdb *memDB) Close() error {
+	mdb.ServicesWrapper = nil
+
+	return nil
+}
+
+func (mdb *memDB) Start() error {
+	mdb.lock.Lock()
+	defer mdb.lock.Unlock()
+
+	return nil
+}
+
+func (mdb *memDB) Commit() error {
+	mdb.lock.Lock()
+	defer mdb.lock.Unlock()
+
+	return nil
+}
+
+func (mdb *memDB) Rollback() error {
+	mdb.lock.Lock()
+	defer mdb.lock.Unlock()
+
+	return nil
 }
```

Comment:
If we cared about in-memory transactionality, we should implement this. I don't think I care enough to.